econml/dml/dml.py

```diff
@@ -29,74 +29,85 @@
 from ..sklearn_extensions.model_selection import WeightedStratifiedKFold
 from ..utilities import (_deprecate_positional, add_intercept,
                          broadcast_unit_treatments, check_high_dimensional,
-                         cross_product, deprecated, fit_with_groups,
+                         cross_product, deprecated,
                          hstack, inverse_onehot, ndim, reshape,
                          reshape_treatmentwise_effects, shape, transpose,
                          get_feature_names_or_default, filter_none_kwargs)
 from .._shap import _shap_explain_model_cate
+from ..sklearn_extensions.model_selection import get_selector, ModelSelector, SingleModelSelector
 
 
-class _FirstStageWrapper:
-    def __init__(self, model, is_Y, featurizer, linear_first_stages, discrete_treatment):
```

Comment:
Also do we no longer featurizer our first stage models?

Reply:
We only ever used featurization in the first stage models to support linear_first_stages, since then we needed to interact phi(X) with [X;W].

Logically, the featurization is only about the final model, the first stage models just get the raw X and W and can fit whatever kind of model they'd like on them, including one that pipelines featurization with any other logic.